src/Test.java

```diff
@@ -9,10 +9,14 @@ public static void main(String[] args)
 		if(answer.equalsIgnoreCase("yes"))
 		{
 			System.out.println("Disgusting.");
+			System.out.println("What is your favorite type of pizza?");
+			String favPizza = input.nextLine();
```

Comment:
You set favPizza to read in the next line of code which is correct and would work.